docs/reference/commandline/cli.md

```diff
@@ -378,6 +378,37 @@ list of root Certificate Authorities.
 
 ## Examples
 
+### <a name="host"></a> Specify daemon host (-H, --host)
+
+You can use the `-H`, `--host` flag to specify one or more sockets to use when
```

Comment:
I don't think "one or more" works here (for the CLI); only works on the daemon (to make it _listen_ on multiple 😅)

So.. hm.. good find actually; looks like we never updated the flag description (I think it was that way because previously that flag was shared between the daemon and the cli);

```
-H, --host list          Daemon socket(s) to connect to
```

Reply:
I updated the CLI help as well, removing the plural